src/components/shared/PlaynetConstants.tsx

```diff
@@ -0,0 +1,7 @@
+enum HeaderSections {
+  intro = 'intro',
+  activities = 'activities',
+  feedback = 'feedback',
+}
+
+export { HeaderSections };
```

Comment:
```suggestion
export enum HeaderSections {
  INTRO = 'intro',
  ACTIVITIES = 'activities',
  FEEDBACK = 'feedback',
}
```

nit: make enum fields caps because they are global